### PR TITLE
Various shortcuts to enhance compression and reduce chain lookups

### PR DESCRIPTION
**Pull request by darkwing**
Some very simply strategies to allow for better minification of x-tag.  Also contains a few improvements to shorten JS lookup chains.

### CODE REVIEW

File: x-tag.js [hunk 1]
```diff
@@ -339,7 +343,7 @@
     * @param {string} value The value of the property.
     */
     defineProperty: function(element, property, accessor, value){
-      return document.documentElement.__defineGetter__ ? 
+      return doc.docElement.__defineGetter__ ?
```
Comment (pennyfx) [inline x-tag.js:346]: find and replace error?  `doc.documentElement`

File: x-tag.js [hunk 2]
```diff
@@ -34,7 +36,7 @@
   * }
   */
   var prefix = (function() {
-    var styles = window.getComputedStyle(document.documentElement, '');
+    var styles = win.getComputedStyle(doc.docElement, '');
```
Comment (pennyfx) [inline x-tag.js:39]: find and replace error?  `doc.documentElement`

File: x-tag.js (hunk 1 above)
Comment (darkwing) [inline x-tag.js:346]: _Sigh_, committed wrong code branch;  lemme fix this

**Comment by darkwing** [comment]
Wrong branch PR'd here. Closing.

**Pull request closed without merging**